modules/installation-gcp-config-yaml.adoc

```diff
@@ -72,23 +70,23 @@ endif::[]
 platform:
   gcp:
     ProjectID: openshift-production <1>
-    region: us-central-1 <1>
+    region: us-central-1a <1>
```

Comment:
us-central-1a is not a valid region for gcp, please use us-central1 instead. Please check out https://cloud.google.com/compute/docs/regions-zones#available for all available regions